WcaOnRails/spec/ruby_version_spec.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe "The ruby version used" do
+  it "matches the one in the .ruby-version file" do
```

Comment:
Sorry for the slippery slope here, but so long as we're doing this, how about checking that .travis.yml is in sync with this as well? Something like this:

```ruby
irb(main):002:0> require 'yaml'; YAML.load(File.read("../.travis.yml"))['rvm']
=> ["2.3.1"]
```

Bonus points for checking our chef script as well: https://github.com/jfly/worldcubeassociation.org/blob/978e64ce3be0600d6feb3fb5e325bf5d910c862a/chef/site-cookbooks/wca/recipes/default.rb#L113. Perhaps change our chef script to read the `.ruby-version` file parse out the minor version number?